go.mod

```diff
@@ -2,6 +2,15 @@ module github.com/rancher/rancher
 
 go 1.22
 
+replace (
+	github.com/rancher/aks-operator => github.com/krunalhinguu/aks-operator v1.29.3-rancher1
+	github.com/rancher/eks-operator => github.com/krunalhinguu/eks-operator v1.29.3-rancher1
+	github.com/rancher/gke-operator => github.com/krunalhinguu/gke-operator v1.29.3-rancher1
+	github.com/rancher/rke => github.com/krunalhinguu/rke v1.29.3-rancher1
+	github.com/rancher/shepherd => github.com/caliskanugur/shepherd v0.0.0-20240507144611-b10827fbb1f1
+	github.com/rancher/steve => github.com/krunalhinguu/steve v1.29.3-rancher1
```

Comment:
Do we still need this fork?

Reply:
@kinarashah please check this commit, it being handled there

https://github.com/rancher/rancher/commit/373a1d6d7bd4b8a00f39a419b4f299a8a0a3f28b